views/ingestion_metrics.view.lkml

```diff
@@ -115,10 +118,10 @@ view: ingestion_metrics {
     sql: ${TABLE}.latency_overflow ;;
   }
 
```

Comment:
Validate with BQ if these fields are deprecated. We can't use connector based Ingestion Metrics as SOT.